Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -391,17 +380,21 @@ final class CodecTests: MongoSwiftTestCase {
             "ts": Timestamp(timestamp: 1, inc: 2),
             "int32": 5,
             "int64": Int64(6),
-            "dec": Decimal128("1.2E+10"),
+            "dec": Decimal128("1.2E+10")!,
             "minkey": MinKey(),
             "maxkey": MaxKey(),
-            "regex": RegularExpression(pattern: "^abc", options: "imx")
+            "regex": RegularExpression(pattern: "^abc", options: "imx"),
+            "symbol": expected.symbol,
```

Comment:
I've moved this whole construction into a `toDocument` method on `AllBSONTypes`. I kept the extJSON string literal below as its own thing since I assume it's defined separately from our extJSON implementation purposefully.

Reply:
yeah, I think that is there to test parsing a doc from ext JSON.